app/services/coordination/document_version_cache.rb

```diff
@@ -10,17 +10,18 @@ def initialize(content_id, payload_version:)
       @payload_version = payload_version
     end
 
-    # Checks whether this document version is outdated (because the cache tracks a newer version).
-    def outdated?
+    # Checks whether this document should be synced based on the current payload version and the
+    # latest synced version in the cache (if any).
+    def sync_required?
       # Sense check: This shouldn't ever come through as nil from Publishing API, but if it does,
       # the only really useful thing we can do is ignore this check entirely because we can't
```

Comment:
Does this comment need to be rewritten now that the logic has been reversed? Before it was saying that the document was up to date with the cached / remote version, so it's not outdated, do nothing.

Now I think it's saying, our payload version is `nil` we don't know how it compares to the cached / remote version, so better sync it to be on the safe side. Is that correct?